test/testsuite/Spec/Random.hs

```diff
@@ -0,0 +1,18 @@
+module Spec.Random (spec) where
+
+import qualified RIO.ByteString as BS
+import           Test.Hspec as Hspec
+
+-- ⚛
+
+import           Fission.Prelude
+import qualified Fission.Random as Random
+
+-- 👩‍🔬
```

Comment:
Same here 🤔

Reply:
Oooh, okay, so this is the label for "tests", because it's an experimenalist running tests 😛 

My comment about needless naming still applies, but this is just _so cute_ that sure let's leave it for now


![](https://media3.giphy.com/media/SmoCFhZCi1kzu/giphy.gif?cid=36b14facf762ddefd5022398b44a960838ff2f3fa59d3457&rid=giphy.gif)

Reply:
Just removed it ;p